lifecycle/README.rst

```diff
@@ -1,11 +1,11 @@
 Introduction
 ------------
 
-ROS2 introduces the concept of managed nodes, also called ``LifecycleNode``\ s.
+ROS 2 introduces the concept of managed nodes, also called ``LifecycleNode``\ s.
 In the following tutorial, we explain the purpose of these nodes, what makes them different from regular nodes and how they comply to a lifecycle management.
 Managed nodes are scoped within a state machine of a finite amount of states.
```

Comment:
I don't think this sentence is very clear. Managed nodes implement a state machine. I don't know what "scoped within a state machine" is referring to and I think that a finite number of states goes without saying.

Reply:
I refactored it in 3556153